testsuites/basic/provisioning.py

```diff
@@ -14,22 +14,32 @@
 
 
 @TestSuiteMetadata(
-    area="core",
+    area="provisioning",
     category="functional",
     description="""
-    This test suite uses to test an environment provisioning correct or not.
+    This test suite uses to verify if an environment can be provisioned correct or not.
+
+    - The basic smoke test can run on all images to determinate if a image can boot and
+    reboot.
+    - Other provisioning tests verify if an environment can be provisioned with special
+    hardware configurations.
     """,
 )
 class Provisioning(TestSuite):
     TIME_OUT = 300
 
     @TestCaseMetadata(
         description="""
-        This test try to connect to ssh port to check if a node is healthy.
-        If ssh connected, the node is healthy enough. And check if it's healthy after
-        reboot. Even not enable to reboot, it's passed with a warning.
-        If the node can be rebooted, but there is panic detected, or not be able to
-        connected. The case failed.
+        This case verifies whether a node is operating normally.
+
+        Steps,
+        1. Connect to TCP port 22. If it's not connectable, failed and check whether
+            there is kernel panic.
+        2. Connect to SSH port 22, and reboot the node. If there is an error and kernel
+            panic, fail the case. If it's not connectable, also fail the case.
+        3. If there is another error, but not kernel panic or tcp connection, pass with
+            warning.
+        4. Otherwise, fully passed.
```

Comment:
Wait, this is *not* what I see written in code. It will try and connect with addr:22, but no SSH protocol involved/tested! It will not exchange any data on that socket, only try to open it. If that connection fails, it tries to read from serial. Then it reboots and reads from serial again on any error.

Reply:
It does connect in `node.reboot()`. It's not explicitly as expected. I will add code comments about it. smoke_test is a very special case, which runs before SSH. Other cases should be more clear.

Reply:
I see, thanks.